### PR TITLE
master-lrqa-49184

### PR DESCRIPTION
**Pull request by Jeremy-Ch**
https://issues.liferay.com/browse/LRQA-49184

### CODE REVIEW

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by Jeremy-Ch** [comment]
ci:test:sf

**Comment by austinchiang** [comment]
'Just <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/BulkEditing.testcase [hunk 1]
```diff
@@ -66,6 +66,111 @@ definition {
...
+
+		Click(locator1 = "Icon#EDIT_CATEGORIES");
+
+		for (var categoryName : list "Books,Plants") {
+			DMDocument.editCategoryTags(categoryName = "${categoryName}");
+
+			Click(locator1 = "Icon#EDIT_CATEGORIES");
+		}
+
+		Uncheck(locator1 = "DocumentsAndMedia#DESCRIPTIVE_LIST_DOCUMENT_CHECKBOX");
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/BulkEditing.testcase]: Can you make a macro for this and apply to other places in the test as well?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/BulkEditing.testcase [hunk 2]
```diff
@@ -66,6 +66,111 @@ definition {
 			searchAssetType = "Document");
 	}
 
+	@description = "This test covers LPS-94956. It ensures that users can delete common category tags by bulk using append action."
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/BulkEditing.testcase]: by using the bulk append action.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/BulkEditing.testcase [hunk 3]
```diff
@@ -66,6 +66,111 @@ definition {
...
+		Navigator.gotoPage(pageName = "Documents and Media Page");
+
+		DMNavigator.gotoDocumentPG(dmDocumentTitle = "Document_2.jpg");
+
+		DMDocument.viewCategoriesPG(categoryNameList = "Pets");
+
+		DMDocument.viewNoCategoriesPG(categoryName = "Books");
+	}
+
+	@description = "This test covers LPS-94955. It ensures that users can edit category tags by bulk using append action."
```
Comment (austinchiang) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/documentmanagement/BulkEditing.testcase]: by using the bulk append action.

**Comment by austinchiang** [comment]
Please resend

**Pull request closed without merging**